source/overview/features.rst

```diff
@@ -0,0 +1,282 @@
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <h4>Mobile applications</h4>
+
+ <img src="https://user-images.githubusercontent.com/29708087/35338058-dbcdda42-00ea-11e8-94af-b6034fababb9.png" height="235px"  width="400px">
+
+.. raw:: html
+
+                    <p>Collaborate with your teams on the go with rich native mobile apps, available for iOS and Android.</p>
+                </td>
+                <td>
+                    <h4>Desktop applications</h4>
+
+ <img src="https://user-images.githubusercontent.com/29708087/35308723-02c2421a-0077-11e8-88c7-b27ee1d4a201.png" height="235px"  width="400px">
+
+.. raw:: html
+
+                    <p>Use the full-featured web interface, or native client apps for Windows, OSX, and Linux.</p>
+                </td>
+            </tr>
+                <td>
+                    <h4>Multi-team support</h4>
+
+ <img src="https://user-images.githubusercontent.com/29708087/35307710-122714c4-0072-11e8-8185-35a794d1889d.png" height="235px"  width="400px">
+
+.. raw:: html
+
+                    <p>Work across multiple teams and projects from the same rich interface.</p>
+                </td>
+                <td>
+                    <h4>Threaded conversations</h4>
+
+ <img src="https://user-images.githubusercontent.com/29708087/35308735-1bc35be6-0077-11e8-830b-e9019158f376.png" height="235px"  width="400px">
+
+.. raw:: html
+
+                    <p>Keep conversations organized and on-point with threading and markdown formatting.</p>
+                </td>
+            <tr>
+                <td>
+                    <h4>Theme colors</h4>
+
+ <img src="https://user-images.githubusercontent.com/29708087/35308755-323cb53e-0077-11e8-82bf-58fcbd777977.png" height="235px"  width="400px">
+
+.. raw:: html
+
+                    <p>Personalize your experience with built-in and custom themes.</p>
+                </td>
+                <td>
+                    <h4>Multi-language support</h4>
+
+ <img src="https://user-images.githubusercontent.com/29708087/35307831-c75b34b0-0072-11e8-88ab-b245b81cfd41.png" height="235px"  width="400px">
+
+.. raw:: html
+
+                    <p>Work across borders with one messaging system translated into over ten languages.</p>
+                </td>
+            </tr>
+        </tbody>
+    </table>
+
+
+
+
+Enterprise Edition Features
+----------------------------------------------
+
+----------------------------------------------
+
+.. raw:: html
+
+    <table class="features__table">
+        <tr>
+            <td>
+                <h4>AD/LDAP SSO</h4>
+
+ <img src="https://user-images.githubusercontent.com/29708087/35307948-46888134-0073-11e8-98f8-18b24f6b1afc.png" height="235px"  width="400px">
+
+.. raw:: html
+
+                <p>
+                    Streamline user provisioning and permissions with directory service integration.
```

Comment:
Saying "permissions" are handled by LDAP might be misleading. It's planned to have LDAP groups integrate with advanced permissions but we don't have it yet.

Reply:
From Chen: "We're saying "permissions are streamlined" by LDAP integration. Is that inaccurate? I was thinking about the directory-sync features specifically: https://docs.mattermost.com/deployment/sso-ldap.html#configure-ad-ldap-synchronization."